src/arduino/components/Pressure.cpp

```diff
@@ -0,0 +1,50 @@
+#include "pressure.h"
+
+/**
+ * Returns pressure in PSI
+ *
+ * LOGIC:
+ * 1. Read sensor value
+ * 2. Convert sensor value to voltage
+ * 3. Convert voltage to pressure
+ * 4. Set pressure
+ * 5. Return pressure
+ */
+float Pressure::getPressure()
+{
+  double sensorRead = analogRead(sensorPin) * 0.0049;                                                                // Volts
```

Comment:
whats the magic number here?

Reply:
I think its noise of the pin but I have no idea

Reply:
my guess based on the equation the voltage of the analog signal is 1/200 of the actual value of the sensor.

Reply:
I asked Ryland and he pointed to this document https://docs.google.com/document/d/1SdptUUIz2HHxYaZK3It0fdqb0d3JrSSpMELT_E1a4ds/edit